src/js/pages/services/ServicesTab.js

```diff
@@ -72,10 +74,18 @@ var ServicesTab = React.createClass({
     });
   },
 
+  handleCloseDeployServiceModal: function () {
+    this.setState({isDeployServiceModalShown: false});
+  },
+
   handleCloseGroupFormModal: function () {
     this.setState({isServiceGroupFormModalShown: false});
   },
 
+  handleOpenDeployServiceModal: function () {
+    this.setState({isDeployServiceModalShown: true});
+  },
+
```

Comment:
This starts to look like a 🍝 . How about we refactor these 4 methods into 1 that accepts a string and a boolean? In any case it makes sense to also toggle the "other" modal when showing this one, as we can't hold 2 modals at once.

Something like 
```
handleOpenModal: function(modalName, show = true) {
   // ...
   if (modalName === "Deploy Group Name Here") {
     this.setState({
       isServiceModalShown: !show, 
       isServiceGroupModalShown: show
    });
 }
```